gspread/utils.py

```diff
@@ -118,6 +118,41 @@ class GridRangeType(StrEnum):
     ListOfLists = "ListOfLists"
 
 
+class ValidationConditionType(StrEnum):
```

Comment:
Could you add a new enum too for `RelativeDate` possible values ?

Documented here: https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/other#relativedate

Reply:
I originally did add an enum for Relative Date but it is not supported by validation requests.